<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkypeExport</class>
 <widget class="QMainWindow" name="SkypeExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Skype Export</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="contactListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWebView" name="htmlViewer" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLocal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>555</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupTimeReference</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonUTC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>575</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>UTC</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupTimeReference</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton24h">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>505</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24h</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupTimeFormat</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton12h">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>485</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>12h</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupTimeFormat</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>465</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time Format</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time Reference</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainWindow">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuConversations">
    <property name="title">
     <string>Conversations</string>
    </property>
    <addaction name="actionExport_Current"/>
    <addaction name="actionExport_All"/>
   </widget>
   <addaction name="menuMainWindow"/>
   <addaction name="menuConversations"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExport_Current"/>
   <addaction name="actionExport_All"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExport_Current">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Current</string>
   </property>
  </action>
  <action name="actionExport_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>qwebview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupTimeFormat"/>
  <buttongroup name="buttonGroupTimeReference"/>
 </buttongroups>
</ui>
